packages/protocol/contracts/libs/LibAddress.sol

```diff
@@ -20,10 +20,23 @@ import { IERC1271Upgradeable } from
 library LibAddress {
     bytes4 private constant EIP1271_MAGICVALUE = 0x1626ba7e;
 
-    /// @dev Wrap this into a new function so the parameter `to` is `address`
-    /// instead of `address payable`.
+    error ETH_TRANSFER_FAILED();
+
+    /// @dev Sends Ether to the specified address. It is recommended to avoid
+    /// using `.transfer()` due to potential reentrancy issues.
+    /// Reference:
+    /// https://consensys.net/diligence/blog/2019/09/stop-using-soliditys-transfer-now
+    /// @param to The recipient address.
+    /// @param amount The amount of Ether to send in wei.
     function sendEther(address to, uint256 amount) internal {
-        AddressUpgradeable.sendValue(payable(to), amount);
+        // Check for zero-value or zero-address transactions
+        if (to == address(0)) return;
```

Comment:
Shall we remove AddressUpgradeable import?

Reply:
We can,but it has the `isContract()` check, which we need - or we can port over.

Reply:
I see, we still need it elsewhere.